docs/reference/api/serving.md

```diff
@@ -6,9 +6,6 @@
 <li>
 <a href="#serving.knative.dev%2fv1alpha1">serving.knative.dev/v1alpha1</a>
 </li>
```

Comment:
Hmm, is this because I left the empty thing?

Reply:
I'm guessing so I ran the tool

Reply:
I can manually drop this